docs/scripts/doc8_style_check.sh

```diff
@@ -2,4 +2,4 @@
 # halt script on error
 set -e
 # Check for Style Code Violations
-doc8 --max-line-length 100 ./docs/source --ignore D000 --quiet
+doc8 --max-line-length 100 source --ignore D000 --quiet
```

Comment:
I think this change is causing the build failures. See: https://travis-ci.org/github/nexB/scancode-workbench/jobs/665056824#L270